logger/middleware/http.go

```
@@ -72,6 +97,30 @@ func HTTPRequestLogger(skipRoutes []string) func(http.Handler) http.Handler {
 	}
 }
 
+func filterBody(body []byte, filterKeys interface{}) map[string]interface{} {
+	if _, ok := filterKeys.([]string); ok {
+		filters := filterKeys.([]string)
+		var b map[string]interface{}
+		_ = json.Unmarshal(body, &b)
```

[suggestion]
Reconsider ignoring the error returned by `Unmarshal` and handle the error properly.

---

Fixing this. (In my head it was already fixed)